spring-cloud-circuitbreaker-spring-retry/src/main/java/org/springframework/cloud/circuitbreaker/springretry/SpringRetryConfig.java

```diff
@@ -39,40 +39,45 @@ boolean isForceRefreshState() {
 		return forceRefreshState;
 	}
 
-	void setForceRefreshState(boolean forceRefreshState) {
```

Comment:
return `this` instead of plain setters. this pattern is used in `SpringRetryConfigBuilder`, but not here (in `SpringRetryConfig`). Let's bring it on par.